src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/RemoveAzureRmServiceFabricNodeType.cs

```diff
@@ -94,12 +94,15 @@ public override void ExecuteCmdlet()
                 if (existingNodeType != null)
                 {
                     cluster.NodeTypes.Remove(existingNodeType);
+
```

Comment:
I have been given feedback in the past to remove code instead of commenting it out like this.

Reply:
@juhacket the reason I left it in is because thers an issue that refers to this piece of code. I also don't know if we are pulling it out permanently or once the issue gets fixed.

Reply:
@praries880 @juhacket correct, we recommend code that is being commented out, but if this piece of functionality is something that we want to bring back in and have an issue tracking the status of its fix, then I think it's fine to keep it in